starport/services/network/path.go

```diff
@@ -27,3 +33,41 @@ func ChainHome(launchID uint64) (string, error) {
 	home, err := os.UserHomeDir()
 	return filepath.Join(home, ChainHomeRoot, strconv.FormatUint(launchID, 10)), err
 }
+
+// GenesisPath returns the default genesis path into the home dir
+func GenesisPath(home string) string {
+	return filepath.Join(home, genesisPath)
+}
+
+// Gentx returns the default gentx path into the home dir
+func Gentx(home string) string {
+	return filepath.Join(home, gentxPath)
+}
+
+// getChainGenesis return the chain genesis path
+func getChainGenesis(home string) (gentx.ChainGenesis, bool, error) {
+	genesisPath := GenesisPath(home)
+	_, err := os.Stat(genesisPath)
+	if os.IsNotExist(err) {
+		return gentx.ChainGenesis{}, false, nil
+	} else if err != nil {
+		return gentx.ChainGenesis{}, false, err
+	}
+	net, err := gentx.ParseGenesis(genesisPath)
+	if err != nil {
+		return gentx.ChainGenesis{}, false, err
+	}
+	return net, true, nil
+}
+
+// CheckGenesisContainsAddress returns true if the address exist into the genesis file
+func CheckGenesisContainsAddress(home, addr string) (bool, error) {
```

Comment:
Can we move this and other cosmos util funcs (getChainGenesis, GenesisPath etc.) to the cosmosutil package?